File: engenharia_de_software_III/atividade_3/Q1/Autor.java
package atividade_3.Q1;

public class Autor {
    private int id;
    private String nome;
    private String email;

    public Autor(int id, String nome, String email) {
        this.id = id;
        this.nome = nome;
        this.email = email;
    }
    
    public String getNome() {
        return nome;
    }

}

File: engenharia_de_software_III/atividade_3/Q1/Autor.ts
export class Autor{
    private id: number;
    private nome: String;
    private email: String;

    constructor (i: number, n: String, e: String) {
        this.id = i;
        this.nome = n;
        this.email = e;
    }
    
    public getNome(): String {
        return this.nome;
    }

}
